### DISCUSSION TITLE
[PSD2] Exercise 1.59

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#127] [Esercizi (A-L)]
**Exercise 1.59** Perform the following additions of unsigned hexadecimal numbers. Indicate whether or not the sum overflows an 8-bit (two hex digit) result.

(a) $22_{16} + 8_{16}$
(b) $73_{16} + 2C_{16}$
(c) $7F_{16} + 7F_{16}$
(d) $C2_{16} + A4_{16}$

**Comment by Elia-Belli** [comment]
**Somma tra Esadecimali senza segno a 8bit**
(a) $22_{16} + 8_{16} = 2A_{16}$
(b) $73_{16} + 2C_{16} = 9F_{16}$
(c) $7F_{16} + 7F_{16} = FE_{16}$
(d) $C2_{16} + A4_{16} = 166_{16}$ (Overflow)

**Comment by Arancialt** [comment]
Il punto b a me viene 9F

**Comment by niccolo-fato** [comment]
![photo_2024-11-15_18-16-31](https://github.com/user-attachments/assets/fc76f188-dff2-4907-bc8b-7190dea9cbc3)
